verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -312,15 +316,12 @@ absl::Status LinterConfiguration::AppendFromFile(
   if (config_or.ok()) {
     RuleBundle local_rules_bundle;
     std::string error;
-    if (local_rules_bundle.ParseConfiguration(*config_or, '\n', &error)) {
-      if (!error.empty()) {
-        std::cerr << "Warnings in parse configuration: " << error << std::endl;
-      }
-      UseRuleBundle(local_rules_bundle);
-    } else {
-      std::cerr << "Unable to fully parse configuration: " << error
-                << std::endl;
+    local_rules_bundle.ParseConfiguration(*config_or, '\n', &error);
+    // Log warnings and errors
+    if (!error.empty()) {
+      std::cerr << error;
```

Comment:
Maybe: if we get an error print out that we use the partial configuration.

Reply:
Thanks for the suggestion!